content/en/docs/how-tos/adding-a-new-secret-to-ci.md

```diff
@@ -4,41 +4,129 @@ date: 2020-12-21T10:08:01-04:00
+in the build farm.
 
 ```yaml
 - from:
-    namespace: "some-namespace"
-    name: "source-some-secret"
+    namespace: "my-project"
+    name: "my-secret"
   to:
-    namespace: "test-credentials"    # the namespace holding all secrets used in a step
-    name: "target-some-secret"       # a unique identifier for your secret
+    namespace: "test-credentials" # the namespace holding all secrets used in a step
+    name: "my-secret"             # a unique identifier for your secret
 ```
 
-Note that we are in the process of turning down the cluster `api.ci` which currently is still used for the secret mirroring process. A new solution for secret management will be in place before `api.ci` is fully removed.
+**Note:** we are in the process of turning down the `api.ci` cluster which currently hosts the source content for the
+secret mirroring process. A new solution for secret management will be in place before `api.ci` is fully removed.
 
 ## Use A Secret In A Job Step
 
-The most common case is to use secrets in a [step](/docs/architecture/step-registry/#step) of a job. In this case, we **require** the user to mirror secrets to `test-credentials` namespace. The pod which runs the step can access the secrets defined in `credentials` stanza of the step definition. see [injecting-custom-credentials](https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-custom-credentials) for details.
+The most common case is to use secrets in a [step](/docs/architecture/step-registry/#step) of a job. In this case, we
+**require** the user to mirror secrets to `test-credentials` namespace. The pod which runs the step can access the secrets
+defined in `credentials` stanza of the step definition. See [the documentation](https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-custom-credentials)
+for details.
 
 ## Use A Secret In Non-Step jobs
 
 {{< alert title="Warning" color="warning" >}}
-This section is used only for the jobs that had existed before [Test Step Registry](/docs/architecture/step-registry/) was introduced and have not yet been converted to multistage tests with steps. It is strongly suggested to use steps for any new jobs.
+This section is used only for the jobs that had existed before [Test Step Registry](/docs/architecture/step-registry/)
+was introduced and have not yet been converted to multistage tests with steps. It is strongly suggested to use steps for
+any new jobs.
 {{< /alert >}}
 
-For non-step jobs, we have to use `ci` as the targeting namespace in secret-mirroring.
+For non-step jobs, we have to use `ci` as the targeting namespace in the secret mirroring configuration.
 
-* For a job which is generated from `ci-operator`'s config and does not use steps, we can mount the secrets via `secrets` stanza in the ci-operator's config, e.g.,
+* For a job which is generated from `ci-operator` configuration and does not use steps, we can mount the secrets via
```

Comment:
NIT: your editor has probably a diff. line breaking policy from mine. :)

Reply:
Yes - yours did not look like it had breaks at all!